pkgs/applications/virtualization/virtualbox/default.nix

```diff
@@ -389,15 +389,18 @@ stdenv.mkDerivation (finalAttrs: {
     ln -s "${finalAttrs.virtualboxGuestAdditionsIso}" "$out/share/virtualbox/VBoxGuestAdditions.iso"
   '';
 
-  preFixup =
+  postFixup =
```

Comment:
This is important, it doesn't seem to matter for `vulkan-loader`, but for `libGL` it does. I believe the explanation is [here](https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/ad/addDriverRunpath/setup-hook.sh#L4).